bgpd/bgp_route.c

```diff
@@ -12318,6 +12318,12 @@ DEFPY(show_ip_bgp, show_ip_bgp_cmd,
 	if (!idx)
 		return CMD_WARNING;
 
+	if (!bgp) {
+		vty_out(vty,
```

Comment:
You are getting bgp NULL in bgp_vty_find_and_parse_afi_safi_bgp ?
but are checking the bgp != NULL after the function call bgp_vty_find_and_parse_afi_safi_bgp(), so how you are avoiding the crash.